swisscovid-cn-sdk-backend/swisscovid-cn-sdk-backend-ws/src/main/java/ch/ubique/swisscovid/cn/sdk/backend/ws/insertmanager/insertfilters/FakeRequestFilter.java

```diff
@@ -20,7 +20,7 @@
       Object principal)
       throws InsertException {
     return uploadVenueInfoList.stream()
-        .filter(uploadVenueInfo -> !uploadVenueInfo.getFake())
+        .filter(uploadVenueInfo -> (uploadVenueInfo.getFake().toByteArray()[0] == 0)) // only collect real venue infos.
```

Comment:
Maybe add a check here to ensure the byte array is non-empty

Reply:
New exception introduced. Loop over infos and check for length. if != 1, throw exception